### DISCUSSION TITLE
mspider/user-guide/sidecar-management/ns-sidecar/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1251] [Announcements]
# mspider/user-guide/sidecar-management/ns-sidecar/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/sidecar-management/ns-sidecar/

**Comment by liyuerich** [comment]
有个清除策略，能介绍一下吗？

**Comment by liyuerich** [comment]
命名空间边车管理的图片需要更新： 没有“边车自动注入”，现在是 “注入策略”

**Comment by liyuerich** [comment]
专有网格和托管网格看到的不太一样，能分开介绍吗？
